include/jemalloc/internal/jemalloc_internal_inlines_a.h

```diff
@@ -14,6 +14,15 @@ malloc_getcpu(void) {
 	return GetCurrentProcessorNumber();
 #elif defined(JEMALLOC_HAVE_SCHED_GETCPU)
 	return (malloc_cpuid_t)sched_getcpu();
+#elif defined(HAVE_RDTSCP)
+	unsigned int dx;
+	asm volatile("rdtscp" : "=d"(dx) :: "memory");
```

Comment:
1. Why does it have the constraint "=d" and not "=r" like the next one?
2. Why does it have "memory" clobber? It doesn't seem to modify anything other than the output operand.

Reply:
1. for this case, we re only interested in (r)dx.
2. You re right it's pointless in this case.